src/Services/etvr/controller.ts

```diff
@@ -30,17 +43,158 @@ export class EyeTrackVrController {
         }
     }
 
-    public getTrackerPosition(position: TRACKER_POSITION): TRACKER_NAME {
-        switch (position) {
-            case TRACKER_POSITION.LEFT_TRACKER: {
-                return TRACKER_NAME.LEFT_EYE
+    public async establishConnection(): Promise<CONNECTION_STATUS> {
+        let tries = 3
+        while (tries > 0) {
+            try {
+                const serverStatus = await this.getServerStatus()
+
+                if (serverStatus === CONNECTION_STATUS.DISCONNECTED) {
+                    await this.startServer()
+                }
+                const status = await this.getServerStatus()
+                return status
+            } catch {
+                if (!tries) {
+                    return CONNECTION_STATUS.FAILED
+                }
```

Comment:
Since you're running `while(tries > 0)` we're never gonna drop here, will we?

Reply:
In the current example, there's a small chance that the number of attempts could reach zero, but at the moment, it stops after just one attempt.